.github/workflows/tests-s3-image.yml

```diff
@@ -43,7 +43,7 @@ on:
         description: 'Publish S3-only images on Dockerhub'
         required: false
         type: boolean
-        default: false
+        default: true
```

Comment:
Quick reminder to remove this before merging the PR :)

Reply:
It actually did not work 😭 Maybe I had 2 conditions, I think there was something up with default values. I'll try something else

Reply:
@alexrashed it seems I can't try it on a branch anymore because the helper script forces it to be on the master branch. I suppose I'll merge this iteration and we can see then? 😅